routes/studentRoutes.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+const db = knex(knexConfig.development);
+
+const responseStatus = {
```

Comment:
I would just import your `responseStatus` object from your other route to make your code DRYer.